model/api/src/main/java/org/jboss/aerogear/unifiedpush/api/Certificate.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.aerogear.unifiedpush.api;
+
+
+public class Certificate extends BaseModel {
+
+    private byte[] certificateData;
+
+    public byte[] getCertificateData() {
```

Comment:
> What about getContent ?
Not sure - naming is hard :-) but to me content implies text, instead of binary.

How about `getRawCertificate()` ?